processor/src/main/java/org/mapstruct/ap/internal/model/source/selector/XmlElementDeclSelector.java

```diff
@@ -69,7 +69,7 @@ public XmlElementDeclSelector(Types typeUtils) {
             }
 
             String name = xmlElementDecl.name().get();
-            TypeMirror scope = xmlElementDecl.scope().get();
+            TypeMirror scope = xmlElementDecl.scope().getValue();
```

Comment:
From the top of my head: get() will return the default value when null,  but in this case we want getValue();

Reply:
So.. to be precise. It does perhaps not make a difference.  Its indeed as above. Omitting scope from the annotation (as is done in the test case) gives  a scope:
1.   `GLOBAL.class` with `get()`
2. `null` with `getValue()`

I prefer the latter one.. It will not go into `typeUtils.isSameType( scope, xmlElementRefInfo.sourceType() )` on line 76

Reply:
Makes sense